Gallery/Panels/NCL_panel_4.py

```diff
@@ -128,7 +131,7 @@ def plot_labelled_filled_contours(data, ax=None):
 
 
 ###############################################################################
-# Plot:
+# Create plot
```

Comment:
I agree here actually, but the previous version is actually pretty consistent across all examples. Maybe this could be a separate PR where we update some section headings?